chain/client/src/client_actor.rs

```diff
@@ -72,21 +71,17 @@ impl ClientActor {
         adv: crate::adversarial::Controls,
         config_updater: Option<ConfigUpdater>,
     ) -> Result<Self, Error> {
-        let state_parts_arbiter = Arbiter::new();
         let self_addr = ctx.address();
         let self_addr_clone = self_addr;
-        let sync_jobs_actor_addr = SyncJobsActor::start_in_arbiter(
-            &state_parts_arbiter.handle(),
-            move |ctx: &mut Context<SyncJobsActor>| -> SyncJobsActor {
-                ctx.set_mailbox_capacity(SyncJobsActor::MAILBOX_CAPACITY);
-                SyncJobsActor {
-                    actions: SyncJobsActions::new(
-                        self_addr_clone.with_auto_span_context().into_multi_sender(),
-                        ctx.address().with_auto_span_context().into_multi_sender(),
-                    ),
-                }
-            },
+
+        let sync_jobs_sender = LateBoundSender::<SyncJobsSenderForSyncJobs>::new();
```

Comment:
that's an interesting way to remove the need for "start_in_arbiter".

Reply:
We still do have start_in_arbiter in `spawn_actix_actor` function call